README.md

```diff
@@ -4,37 +4,680 @@ Sylvia is the old name meaning Spirit of The Wood.
+
+## Using not implemented entry points
+
+Sylvia is not yet implementing all the possible CosmWasm entry points, and even
+when it will - it might happen that some will be added in the future, and Sylvia
+would not align immediately. Hopefully, you can always use traditional entry points
+for anything which is not implemented - for example, IBC calls. As an example, let's
+see how to implement replies for messages:
+
+```rust
+use sylvia::cw_std::{DepsMut, Env, Reply, Response};
+
+#[contract(error = ContractError)]
+#[messages(group as Group)]
+impl MyContract<'_> {
+    fn reply(&self, deps: DepsMut, env: Env, reply: Reply) -> Result<Response, ContractError> {
+        todo!()
+    }
+    // Some items defined previously
+}
+
+#[entry_point]
+fn reply(deps: DepsMut, env: Env, reply: Reply) -> Result<Response, ContractError> {
+    CONTRACT.reply(deps, env, reply)
+}
+```
+
+It is important to create an entry function in the contract type - this way, it
+gains access to all the state accessors defined on the type.
+
+## Multitest
+
+Sylvia also generates some helpers for testing contracts - it is hidden behind the
+`mt` feature flag, which has to be enabled.
+
+It is important to ensure no `mt` flag is set when the contract is built in `wasm`
+target because of some dependencies it uses, which are not buildable on Wasm. My
+recommendation is to add an additional `sylvia` entry with `mt` enabled in the
+`dev-dependencies`, and also add the `mt` feature on your contract, which enables
+mt utilities in other contract tests. An example `Cargo.toml`:
+
+```rust
+[package]
+name = "my-contract"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[features]
+library = []
+mt = ["sylvia/mt"]
+
+[dependencies]
+cosmwasm-schema = "1.2.5"
+cosmwasm-std = "1.2.5"
+cw-storage-plus = "1.0.1"
+schemars = "0.8.12"
+serde = "1.0.160"
+sylvia = "0.3.2"
+thiserror = "1.0.40"
+
+[dev-dependencies]
+sylvia = { path = "0.3.2", features = ["mt"] }
+```
+
+There would obviously be more dependencies - most probably `cw-strorage-plus`,
+but this is just to show how I enable the `mt` flag. With that, we can use mt
+utils in the contract:
+
+```rust
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use sylvia::multitest::App;
+
+    #[test]
+    fn counter_test() {
+        let app = App::default();
+
+        let owner = "owner";
+
+        let code_id = contract::CodeId::store_code(&app);
+
+        let contract = code_id.instantiate(3)
+            .with_label("My contract")
+            .call(owner)
+            .unwrap();
+
+        let counter = contract.counter().unwrap();
+        assert_eq!(counter, contract::CounterResp { counter: 3});
+
+        contract.increment().call(owner).unwrap();
+
+        let counter = contract.counter().unwrap();
+        assert_eq!(counter, contract::CounterResp { counter: 4});
+    }
+}
+```
+
+First of all, note the `contract` module I am using here - it is a slight change
```

Comment:
Very nice explanation of multi-test usage

Reply:
Unfortunately not true before 0.4 because of #108 has to be first merged.